Gems/AWSGameLift/Code/AWSGameLiftClient/Source/Activity/AWSGameLiftJoinSessionActivity.cpp

```diff
@@ -44,14 +44,24 @@ namespace AWSGameLift
         {
             Multiplayer::SessionConnectionConfig sessionConnectionConfig;
             auto createPlayerSessionResult = createPlayerSessionOutcome.GetResult();
-            // TODO: AWSNativeSDK needs to be updated to support this attribute, and it is a must have for TLS certificate enabled fleet
-            //sessionConnectionConfig.m_dnsName = createPlayerSessionResult.GetPlayerSession().GetDnsName().c_str();
-            sessionConnectionConfig.m_ipAddress = createPlayerSessionResult.GetPlayerSession().GetIpAddress().c_str();
+            AZStd::string_view dnsName = createPlayerSessionResult.GetPlayerSession().GetDnsName().c_str();
+            AZStd::string_view ipAddress = createPlayerSessionResult.GetPlayerSession().GetIpAddress().c_str();
+            // When connecting to a game session that is running on a TLS-enabled fleet, you must use the DNS name, not the IP address.
```

Comment:
Nit: May worth throwing an error if both of DNS name and IP address are specified? Also could you define a constant for the hard coded string below (.amazongamelift.com)?

Reply:
DNS name and ip address are always presented no matter it is tls/non tls use case. Just dns name will have different format for different use case.
I can create a constant for it.